getting-started/Installation/nuget-packages.md

```diff
@@ -48,45 +53,38 @@ There are other Telerik NuGet Packages for the UI suites. The following are the
 
 ## Packages for .NET Standard 2.0
 
-Telerik Document Processing supports [.NET Standard 2.0](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md). The available packages are: 
+The following list represents the available NuGet packages for the Document Processing libraries that you can use with [.NET Standard 2.0](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md):
 
 * __Telerik.Documents.Core:__ The main NuGet package from the Telerik Document Processing libraries. Required when you plan to use the library.
-
 * __Telerik.Documents.Fixed:__ The package is required when processing `PDF` documents.
 * __Telerik.Documents.ImageUtils:__ This package is required when exporting to `PDF` format a document containing images different than Jpeg and Jpeg2000 or ImageQuality different than High. For more information check the [PdfProcessing`s Cross-Platform Support]({%slug radpdfprocessing-cross-platform%}) article.
 * __Telerik.Documents.CMapUtils:__ Provides a default implementation for getting the data of a predefined CMap table by a given name.
-
 * __Telerik.Documents.Flow:__ The package is part of the Words Processing Library. Required for processing `HTML`, `DOCX`, `RTF`, and `TXT` documents.
 * __Telerik.Documents.Flow.FormatProviders.Pdf:__ The package for exporting flow documents, such as `DOCX` and `RTF`, to PDF.
-
 * __Telerik.Documents.Spreadsheet:__ The main package for generating and editing spreadsheets.
 * __Telerik.Documents.Spreadsheet.FormatProviders.OpenXml:__ This package is required for the import/export of `XLSX` documents.
 * __Telerik.Documents.Spreadsheet.FormatProviders.Xls:__ This package is required for the import/export of `XLS` documents.
 * __Telerik.Documents.Spreadsheet.FormatProviders.Pdf:__ Required for the export of RadSpreadProcessing documents to `PDF`.
-
 * __Telerik.Documents.SpreadsheetStreaming:__ The main package for generating huge spreadsheet files while using minimum resources.
-
 * __Telerik.Zip:__ The package is required when working with zipped formats, such as `DOCX` and `XLSX`. It can be used as a standalone library as well. 
 
-
 ## Get the NuGet Packages
 
-You need to add the Telerik package server (https://nuget.telerik.com/nuget) to the package sources in Visual Studio. 
+Before you can install Telerik NuGet packages, you must configure the Telerik NuGet server (https://nuget.telerik.com/nuget) as a package source in Visual Studio:
```

Comment:
The changes in these step-by-step instructions try to make all list items parallel and to add more detail.